src/System.Globalization.Calendars/tests/Misc/Calendars.cs

```diff
@@ -60,59 +46,24 @@ public static void JapaneseTest()
         }
 
         [Fact]
-        public static void JapaneseLunisolarTest()
-        {
-            JapaneseLunisolarCalendar cal = new JapaneseLunisolarCalendar();
-            Assert.True(cal.Eras.Length >= 2);
-        }
-
-        [Fact]
         public static void JulianTest()
         {
             JulianCalendar jc = new JulianCalendar();
             Assert.False(jc.IsLeapYear(1999));
         }
 
         [Fact]
-        public static void KoreanTest()
-        {
-            KoreanCalendar kc = new KoreanCalendar();
-            Assert.Equal(1, kc.Eras.Length);
-        }
-
-        [Fact]
         public static void KoreanLunisolarTest()
         {
             KoreanLunisolarCalendar kls = new KoreanLunisolarCalendar();
             Assert.Equal(1, kls.Eras.Length);
         }
 
         [Fact]
-        public static void PersianTest()
-        {
-            PersianCalendar pc = new PersianCalendar();
-            Assert.Equal(1, pc.Eras.Length);
```

Comment:
Ah, nevermind, I see you added it in this PR at https://github.com/dotnet/corefx/pull/7743/files#diff-581f48459778b4a7e1b5a20f05e2496aR143